tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -11675,5 +11677,25 @@ bool SpirvEmitter::spirvToolsLegalize(std::vector<uint32_t> *mod,
   return optimizer.Run(mod->data(), mod->size(), mod, options);
 }
 
+SpirvInstruction *
+SpirvEmitter::doUnaryExprOrTypeTraitExpr(const UnaryExprOrTypeTraitExpr *expr) {
+  // TODO: We support only `sizeof()`. Support other kinds.
+  if (expr->getKind() != clang::UnaryExprOrTypeTrait::UETT_SizeOf) {
+    emitError("expression class '%0' unimplemented", expr->getExprLoc())
+        << expr->getStmtClassName() << expr->getSourceRange();
```

Comment:
the `<<` operator will pass a string to the the `%X` in the first argument. for example `%0` will be replaced with the string value of ` expr->getStmtClassName() `. 

There is no `%1` here, but there is a second `<<` operator. So `<< expr->getSourceRange()` should be removed. 

expr->getExprLoc() will do the job of pointing to the right place in the error message.